.github/workflows/tmt-tests.yml

```diff
@@ -0,0 +1,51 @@
+---
+name: TMT Tests
+
+on:
+  push:
+    branches: 
+      - main
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      plan_filter:
+        description: 'Plan filter name, ie: tag:smoke'
```

Comment:
'Plan filter name, ie: tag:smoke' this feels odd, can we rename it to a more appropriate description?

Reply:
included a better description.